src/index.html

```diff
@@ -5,8 +5,363 @@
+          </div>
+
+          <a
+            class="card__link"
+            href="#"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+
+        <div class="card__content">
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__description">Product code: 195434</p>
+
+          <div class="card__review-container">
+            <ul class="card__stars card__stars--4">
+              <li class="card__star"></li>
+              <li class="card__star"></li>
+              <li class="card__star"></li>
+              <li class="card__star"></li>
+              <li class="card__star"></li>
+            </ul>
+
+            <p class="card__review-text">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-number">$2,199</p>
+          </div>
+
+          <a
+            class="card__link"
+            href="#"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+
+        <div class="card__content">
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__description">Product code: 195434</p>
+
+          <div class="card__review-container">
+            <ul class="card__stars card__stars--4">
+              <li class="card__star"></li>
+              <li class="card__star"></li>
+              <li class="card__star"></li>
+              <li class="card__star"></li>
+              <li class="card__star"></li>
+            </ul>
+
+            <p class="card__review-text">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-number">$2,199</p>
+          </div>
+
+          <a
+            class="card__link"
+            href="#"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+
+        <div class="card__content">
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__description">Product code: 195434</p>
+
+          <div class="card__review-container">
+            <ul class="card__stars card__stars--4">
```

Comment:
Currently, it's written without mistake since you've written stars as an element of the `card` block in CSS and HTML (so I'll approve - other issues are fixed).

But want to add a comment FYI: you _can_ use `stars` block as an element of another block (BEM-mix), for example (schematically):

```
<block1>
  <block1__element1>
    <block__element2 />
  </block1__element1>

  <block2 block1__block2>
    <block2__element1 />
    <block2__element2 />
  </block2 block1__block2>
</block1>
```
it's allowed :arrow_up: `block1__block2` - it's BEM mix